pkg/config/config.go

```diff
@@ -65,6 +65,11 @@ type JiraConfig struct {
 	AllowInsecure bool
 	Token         string
 	Query         string
```

Comment:
I don't think this field is needed anymore with how we're handling ticket updates. Thoughts?